go/exosecurity/message_authorizer.go

```diff
@@ -0,0 +1,6 @@
+package exosecurity
+
+import "github.com/Originate/exocom/go/structs"
+
+// MessageAuthorizer auhorizes the message
```

Comment:
Lets give this more of an explanation.

Maybe something like "is a function which returns whether or not a message is authorized"